{"name": "После оформления заказа его номер появляется в разделе В работе", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <tests.test_order_feed_page.TestOrderFeedPage object at 0x000001D8A2E67A30>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"17dd8f65e558e45005f581ac9fa41f51\")>\n\n    @allure.title('После оформления заказа его номер появляется в разделе В работе')\n    def test_num_order_in_work(self, driver):\n>       result = OrderFeedPage(driver).base_check_number_order_in_work()\n\ntests\\test_order_feed_page.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages\\order_feed_page.py:265: in base_check_number_order_in_work\n    num_new_order = self.base_check_number_new_order()\npages\\order_feed_page.py:184: in base_check_number_new_order\n    self.wait__element_number_orders()\npages\\order_feed_page.py:52: in wait__element_number_orders\n    return self.wait_element_(locator, text_value)\npages\\base_page.py:43: in wait_element_\n    return WebDriverWait(self.driver, 15).until(expected_conditions.none_of(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"17dd8f65e558e45005f581ac9fa41f51\")>, method = <function none_of.<locals>.none_of_condition at 0x000001D8A2FDB640>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nvenv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Общие шаги: после оформления заказа его номер появляется в разделе \"В работе\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 265, in base_check_number_order_in_work\n    num_new_order = self.base_check_number_new_order()\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 184, in base_check_number_new_order\n    self.wait__element_number_orders()\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 52, in wait__element_number_orders\n    return self.wait_element_(locator, text_value)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 43, in wait_element_\n    return WebDriverWait(self.driver, 15).until(expected_conditions.none_of(\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Общие шаги получения номера нового заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 184, in base_check_number_new_order\n    self.wait__element_number_orders()\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 52, in wait__element_number_orders\n    return self.wait_element_(locator, text_value)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 43, in wait_element_\n    return WebDriverWait(self.driver, 15).until(expected_conditions.none_of(\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Общие шаги для авторизации", "status": "passed", "steps": [{"name": "Ожидаем кликабельность кнопки \"Личный кабинет\"", "status": "passed", "start": 1704912142654, "stop": 1704912142678}, {"name": "Нажимаем на кнопку \"Войти в аккаунт\" на главной странице", "status": "passed", "start": 1704912142678, "stop": 1704912142731}, {"name": "Нажимаем на поле \"Email\" на странице авторизации", "status": "passed", "start": 1704912142738, "stop": 1704912142788}, {"name": "Вводим email зарегистрированного пользователя на странице авторизации", "status": "passed", "start": 1704912142789, "stop": 1704912142859}, {"name": "Нажимаем на поле \"Пароль\" на странице авторизации", "status": "passed", "start": 1704912142859, "stop": 1704912142908}, {"name": "Вводим пароль зарегистрированного пользователя на странице авторизации", "status": "passed", "start": 1704912142908, "stop": 1704912142947}, {"name": "Ожидаем кнопку \"Войти\" на странице авторизации", "status": "passed", "start": 1704912142947, "stop": 1704912142968}, {"name": "Нажимаем на кнопку \"Войти\" на странице авторизации", "status": "passed", "start": 1704912142968, "stop": 1704912143005}], "start": 1704912142649, "stop": 1704912143008}, {"name": "Перетаскиваем ингредиент в заказ", "status": "passed", "start": 1704912143008, "stop": 1704912144128}, {"name": "Нажимаем на кнопку \"Оформить заказ\"", "status": "passed", "start": 1704912144128, "stop": 1704912144177}, {"name": "Ожидание появления нового номера заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 52, in wait__element_number_orders\n    return self.wait_element_(locator, text_value)\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\pages\\base_page.py\", line 43, in wait_element_\n    return WebDriverWait(self.driver, 15).until(expected_conditions.none_of(\n  File \"C:\\Users\\123\\PycharmProjects\\Diplom\\Diplom_3\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1704912144179, "stop": 1704912159694}], "start": 1704912142649, "stop": 1704912159694}], "start": 1704912142649, "stop": 1704912159695}], "start": 1704912142649, "stop": 1704912159697, "uuid": "3fc57387-5b89-4b4e-ad0c-3ac09750bd3f", "historyId": "cd04650ce339822367ca8a65a24d3021", "testCaseId": "cd04650ce339822367ca8a65a24d3021", "fullName": "tests.test_order_feed_page.TestOrderFeedPage#test_num_order_in_work", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed_page"}, {"name": "subSuite", "value": "TestOrderFeedPage"}, {"name": "host", "value": "LAPTOP-588797DN"}, {"name": "thread", "value": "20352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed_page"}]}